### ISSUE TITLE
Unable to install source distribution in fresh environment

### ISSUE THREAD

**Issue by soxofaan** [WFP-VAM/vam.whittaker#4]
setup.py has a dependency on numpy:

https://github.com/WFP-VAM/vam.whittaker/blob/019635568a9ef5b1b84555b2661cdf456277f591/setup.py#L7

which makes it difficult to install a source distribution of vam.whittaker in a fresh environment (without numpy)

e.g. version 2.0.2 has no wheel distribution for python 3.10: https://pypi.org/project/vam.whittaker/2.0.2/#files
Docker based example of trying to install it:
```
$ docker run --rm -it python:3.10 /bin/bash

root@f0a2ef21b90e:/# pip install "vam.whittaker==2.0.2"
Collecting vam.whittaker==2.0.2
  Downloading vam.whittaker-2.0.2.tar.gz (25 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-4a150zqr/vam-whittaker_11498c8e2e244ebe9d9cc039263f3f62/setup.py", line 7, in <module>
          import numpy
      ModuleNotFoundError: No module named 'numpy'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.

```

**Comment by soxofaan** [comment]
Workaround is apparently to install numpy and cython (per #2) first:
```
pip install numpy cython
```

**Comment by valpesendorfer** [comment]
fixed with #6
